backends/aerospike.go

```diff
@@ -68,6 +68,11 @@ func NewAerospikeBackend(cfg config.Aerospike, metrics *metrics.Metrics) *Aerosp
 		clientPolicy.IdleTimeout = time.Duration(cfg.ConnIdleTimeoutSecs) * time.Second
 	}
 
+	// If set, specify the size of the Connection Queue cache per node.
```

Comment:
**Nitpick:** Can you please update this comment to be
```
Aerospike's default connection queue size per node is 256. 
If cfg.ConnQueueSize is greater than zero, it will override the default.
```